examples/spine/src/app.coffee

```diff
@@ -57,23 +57,25 @@ class TodoApp extends Spine.Controller
 		@addNew todo for todo in @getByFilter()
 
 	toggleAll: (e) ->
+		checked = e.target.checked
```

Comment:
The "mark all complete" checkbox can change during the loop below so the initial value needs to be cached.